LICENSE

```diff
@@ -413,17 +391,13 @@ Font-Awesome
 ----------------------
 https://github.com/FortAwesome/Font-Awesome
 
-License
-
-The Font Awesome font is licensed under the SIL OFL 1.1:
-http://scripts.sil.org/OFL
-Font Awesome CSS, LESS, and Sass files are licensed under the MIT License:
-https://opensource.org/licenses/mit-license.html
-The Font Awesome documentation is licensed under the CC BY 3.0 License:
-http://creativecommons.org/licenses/by/3.0/
-Attribution is no longer required as of Font Awesome 3.0, but much appreciated:
-Font Awesome by Dave Gandy - http://fontawesome.io
-Full details: http://fontawesome.io/license/
+Font Awesome Free License
+Font Awesome Free is free, open source, and GPL friendly. You can use it for
```

Comment:
less is more. Could we just emphasize copyright here?

Reply:
The license info is copied from https://fontawesome.com/license/free